moped-editor/src/components/FileUpload/FileUpload.js

```diff
@@ -83,9 +74,12 @@ const FileUpload = props => {
    * @return {Promise<boolean>}
    */
   const handleBeforeAdd = item => {
+    // strip out & because of file upload bug
+    // https://github.com/cityofaustin/atd-data-tech/issues/11900#issuecomment-1505701452
+    const filteredFilename = item.filename.replace("&", "")
```

Comment:
@chiaberry could you replace `&` with `_` instead? that leaves the filename a bit more readable.